src/core/RMXRemixer.m

```diff
@@ -202,4 +197,21 @@ + (void)updateVariable:(RMXVariable *)variable fromRemoteControllerToValue:(id)v
   }
 }
 
+#pragma mark - Remote controller
+
++ (nullable NSURL *)remoteControllerURL {
+  return [[[self sharedInstance] remoteController] remoteControllerURL];
+}
+
++ (BOOL)isSharing {
+  if (![[self sharedInstance] remoteController]) {
```

Comment:
Can you skip this `if` clause and simply return `return [[[self sharedInstance] remoteController] isSharing];` ?

And if not, then won't `applicationDidBecomeActive` and `...resign` fail as well?